src/Tuner.cpp

```diff
@@ -45,16 +45,31 @@
 #include <cblas.h>
 #endif
 
+#ifdef USE_HALF
+const auto TUNER_FILE_LOCAL = std::string("leelaz_opencl_tuning_half");
+constexpr auto MAX_ERROR = 1e-2f;
```

Comment:
Ouch! Guess it's more than enough to detect driver breakage regardless.

You could consider using the same file and just storing it under the "XgemmBatchedHalf" key in the file.